lib/pages/splash/splash_screen.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:structure_flutter/core/resource/text_style.dart';
+
+class SplashScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
```

Comment:
```suggestion
    return SafeArea(
```
i think SplashScreen without SafeArea will look better than